docs/reference/fleet/apis/get-global-checkpoints.asciidoc

```diff
@@ -0,0 +1,61 @@
+[role="xpack"]
+[[fleet-get-global-checkpoints]]
+=== Get global checkpoints API
+++++
+<titleabbrev>Get global checkpoints</titleabbrev>
+++++
+
+The purpose of the get global checkpoints api is to return the current global
+checkpoints for an index. This API allows users to know the what sequence numbers
+have been safely persisted in Elasticsearch.
+
+The API has an optional polling mode enabled by the `wait_for_advance` query
+parameter. In polling mode, the API will only return after the global checkpoints
+advance past the provided `current_checkpoints`. By default, `current_checkpoints`
+is an empty array, which will lead to the API returning immediately.
+
+Currently the `wait_for_advance` parameter is only supported for one shard indices.
+
+[[get-global-checkpoints-api-request]]
+==== {api-request-title}
+
+`GET /<index>/_fleet/global_checkpoints`
+
+[[get-global-checkpoints-api-path-params]]
+==== {api-path-parms-title}
+
+`<index>`::
+(Required, string)
+A single index or index alias that resolves to a single index.
+
+[role="child_attributes"]
+[[get-global-checkpoints-api-query-parms]]
+==== {api-query-parms-title}
+
+`wait_for_advance`::
+(Optional, Boolean) A boolean value which controls whether to wait (until the
+`poll_timeout`) for the global checkpoints to advance past the provided
+`current_checkpoints`.
+
+`current_checkpoints`::
+(Optional, list) A comma separated list of previous global checkpoints.
+When used in combination with `wait_for_advance`, the API will only return once
+the global checkpoints advances past the `current_checkpoints`. Defaults to an
+empty list which will cause Elasticsearch to immediately return the global
```

Comment:
If we rename to previous, the text here could say "...return the current global checkpoints".